tools/gromacs/macros.xml

```diff
@@ -46,13 +46,17 @@
                     <option value="Reaction-Field-zero">Reaction field electrostatics.</option>
                 </param>
                 
-                <param argument="temperature" type="integer" label="Temperature /K" value="0" min="0" max="1000000" help="Temperature" />
+                <param name="temperature" type="integer" label="Temperature (K)" value="0" min="0" max="1000000" help="Temperature" />
+                   <param name="systemTcouple" type="select" label="Number of groups to set for thermocoupling">
+                       <option value="true">Single coupling group (System)</option>
+                       <option value="false" selected="true">Two coupling groups (Protein and Non-Protein)</option>
+                   </param>
                 <param argument="step_length" type="float" label="Step length in ps" value="0" min="0.0001" max="1.0" help="Step length in ps." />
                 <param argument="write_freq" type="integer" label="Number of steps that elapse between saving data points (velocities, forces, energies)" value="0" min="0" max="1000000" help="Step length in ps." />
                 <param argument="rcoulomb" value="1.0" type="float" label="Distance for the Coulomb cut-off."/>
                 <param argument="rlist" value="1.0" type="float" label="Cut-off distance for the short-range neighbor list. Ignored if the Verlet cutoff scheme is set."/>
                 <param argument="rvdw" value="1.0" type="float" label="Short range van der Waals cutoff."/>
-                <param argument="md_steps" type="integer" label="Number of steps for the simulation" value="0" min="0" max="1000000" help="NPT steps" />
```

Comment:
I think there should be some maximum, otherwise we risk wasting a lot of resources. It would be very nice if it could be configured to be overridden by trusted users, but I don't think Galaxy supports that.

Reply:
Ok I get that. Would you be ok with something more reasonable than 1 million steps though? I'm thinking 25 million steps, this gives users 50 nanoseconds at a 2 fs time step?

Reply:
Yes, 25 millions sounds reasonable :+1:

Reply:
If this is not a limitation of the underlying tool, I don't think we should enforce a limit on the userbase. If I am running a Galaxy instance on my own hardware, having a hard coded limit here is not reasonable and would prevent this tool from being actually useful in production instances. 

Providing help text that suggests, not going too high with this value, would be reasonable though.

Any limits related to job resource concerns should be implemented at the job scheduling / administration level.

Reply:
Yes, I agree. I just need to work out how we configure that on EU.